src/ontology/genepio-edit.owl

```diff
@@ -687,7 +693,7 @@ Declaration(Class(obo:GENEPIO_0001116))
 Declaration(Class(obo:GENEPIO_0001117))
 Declaration(Class(obo:GENEPIO_0001118))
 Declaration(Class(obo:GENEPIO_0001119))
-Declaration(Class(obo:GENEPIO_0001120))
```

Comment:
We probably have to deprecate this term as a result of not doing the robot_pha4ge_import.owl file? https://www.ebi.ac.uk/ols/ontologies/genepio/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FGENEPIO_0001120